project.clj

```diff
@@ -7,9 +7,21 @@
                  [instaparse/instaparse "1.4.10"]]
   :profiles {:dev {:dependencies [[criterium/criterium "0.4.5"] ;; benchmark
                                   [org.clojure/test.check "0.10.0"]]
-                   :plugins      [[jonase/eastwood "0.3.5"]]}}
+                   :plugins      [[jonase/eastwood "0.3.5"]]}
+             :provided {:dependencies [[org.clojure/clojurescript "1.10.520"]]}}
   :test-selectors {:default     (fn [m] (not (some #{:benchmark} (keys m))))
                    :benchmark   :benchmark}
+  :cljsbuild {:builds
+              {"dev" {:source-paths ["src" "test"]
+                      :compiler {:main parcera.test-runner
+                                 :output-to "target/out/tests.js"
+                                 :output-dir "target/out"
+                                 :target :nodejs
+                                 :optimizations :none}}}
+              :test-commands
+              {"dev" ["node" "target/out/tests.js"]}}
+  :plugins [[lein-cljsbuild "1.1.7"]]
+  :hooks [leiningen.cljsbuild]
```

Comment:
is it possible to move these to the `dev` profile above ? I am not sure what the behavior is when having two separate places for the same thing 🤔

Reply:
Did that. Had to move some dependencies to the global scope for now though. Not sure what best practice is, but there is this reality: https://clojure.atlassian.net/browse/CLJS-1792

Reply:
Might be relevant:

https://github.com/clojure/clojurescript/commit/731be5e0916ad8d619c302bfc9b985c4d10daa8d